src/DDTrace/SpanContext.php

```diff
@@ -174,6 +126,13 @@ public function getAllBaggageItems()
 
     public function isEqual(SpanContextInterface $spanContext)
     {
+        if (is_subclass_of($spanContext, "DDTrace\Data\SpanContext")) {
```

Comment:
We can remove a function invocation here. :)

```suggestion
        if ($spanContext instanceof SpanContextData) {
```